review/static/js/src/review.js

```diff
@@ -0,0 +1,62 @@
+/* Javascript for ReviewXBlock. */
+function ReviewXBlock(runtime, element) {
+  $(function ($) {
+    'use strict'
+
+    $('.review-button').on('click', function(event) {
+      var $btn = $(event.currentTarget),
+          $content = $btn.siblings('.review-content'),
+          $iframe = $content.find('iframe'),
+          title,
+          skipLink;
+
+      if (!$btn.hasClass('disable-click')) {
+        // Prevents double clicking which caused issues from seeing the problems
+        $btn.addClass('disable-click');
+
+        setTimeout(function() {
+          $btn.removeClass('disable-click');
+        }, 100);
```

Comment:
Instead of setting a timeout, would it be possible to wait until the first click completes whatever it needs to complete first and then removing the class?

Reply:
Nope. This was tested and did not work. It has do with loading the review xblock in "edit" in studio and it adding in another listener.